.github/workflows/extended_tests.yml

```diff
@@ -0,0 +1,16 @@
+# SPDX-FileCopyrightText: 2019, 2020 Alvar Penning
```

Comment:
Last time I checked my calendar it was 2021 and I guess you wrote this, @adur1990?

Reply:
Dang it. This happens if you copy-paste stuff :D

Reply:
I feel you.